### ISSUE TITLE
[Accessibility] the group of Nav that have scrollable content cannot be accessible by keyboard

### ISSUE THREAD

**Issue by pingpingann** [microsoft/fluentui#19432]
### Environment Information

- **Package version(s)**: 7.170.3
- **Browser and OS versions**: Windows 10 + Microsoft Edge version 93.0.926.0

### Describe the issue:

Set the maxHeight for groupContent of Nav, it will be scrollable when the content list is long. the Nav group that have scrollable content cannot be accessible by keyboard.

Note: 
Expand the button of the group and use the Accessibility Insights tool to detect the accessibility bug:
Element path: .is-expanded.ms-Nav-group:nth-child(1) > .ms-Nav-groupContent

### Please provide a reproduction of the issue in a codepen:

codepen: https://codepen.io/pingpingann/pen/abWgNYG

#### Actual behavior:

Elements that have scrollable content are not accessible by keyboard and can't pass accessibility testing

#### Expected behavior:

Elements that have scrollable content can be accessible by keyboard and can pass accessibility testing

### Documentation describing expected behavior

scrollable-region-focusable - https://accessibilityinsights.io/info-examples/web/scrollable-region-focusable

**Comment by gouttierre** [comment]
@pingpingann  - Thanks for filing this issue with us. To set expectation our team will have a look at the issue but only once capacity allows.
  
@bsunderhus -  Would you be able to confirm if this is a regression, or if this behavior is an issue?
cc @smhigley
